### DISCUSSION TITLE
How to install version from specific GH branch/commit

### DISCUSSION THREAD

**Discussion by mvorisek** [mlocati/docker-php-extension-installer#521] [Q&A] [answered]
Docs https://github.com/mlocati/docker-php-extension-installer#installing-specific-versions-of-an-extension does not mention this.

For experimental php support, I need to use non-released version of xdebug from https://github.com/xdebug/xdebug.

Is there any way how I can specify branch or a commit sha as a version for install?

**Comment by mlocati** [comment, marked as answer]
We already have an open issue for this https://github.com/mlocati/docker-php-extension-installer/issues/493
